projects/platformdirs/fuzz_dirs.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import atheris
+
+import platformdirs
+
+
+def TestOneInput(data):
+    fdp = atheris.FuzzedDataProvider(data)
+
+    targets = [
+        platformdirs.user_data_dir,
```

Comment:
Not sure how useful it is to fuzz this. Are there better alternatives from the list to fuzz?

Reply:
There might be some, although it's getting sparse. By and large, the remaining are either similar from a fuzzing perspective to this library, e.g. small in size, or have some challenges being integrated (for the larger projects). I can do some of the other ones and then you can choose!